components/brave_wallet/browser/polkadot/polkadot_tx_meta.h

```diff
@@ -27,6 +28,11 @@ class PolkadotTxMeta : public TxMeta {
   base::Value::Dict ToValue() const override;
   mojom::TransactionInfoPtr ToTransactionInfo() const override;
   mojom::CoinType GetCoinType() const override;
+
+ private:
+  std::string recipient_;
+  std::string encoded_extrinsic_;
+  uint64_t amount_ = 0;
```

Comment:
Let's start with StrictNumeric for new instaces of amounts, coin values, fees, etc.